app/scripts/modules/core/src/pipeline/executions/executionGroup/executionGroup.less

```diff
@@ -127,3 +127,12 @@
   display: block;
   margin-bottom: 5px;
 }
+
+.account-tag-name {
+  max-width: 250px;
+  span {
```

Comment:
Can we make a more specific classname on the span? I want to prevent any unintentional side-effects.